FAST-LIO/CMakeLists.txt

```diff
@@ -66,13 +71,22 @@ include_directories(
   ${EIGEN3_INCLUDE_DIR}
   ${PCL_INCLUDE_DIRS}
   ${PYTHON_INCLUDE_DIRS}
+  ${GTSAM_INCLUDE_DIR}
+  ${GeographicLib_INCLUDE_DIRS}
   include)
 
 add_message_files(
   FILES
   Pose6D.msg
 )
 
+add_service_files(
```

Comment:
When needed, service files should be added not in the repo itself, but in the other repo called ros-messages-88, but in this case this service shouldn't be needed because you can use the 'save_pcd' arg directly in decco.launch

Reply:
Ooh nice. Removed!